### PR TITLE
Small html improvements

### PR DESCRIPTION
**Pull request by mukrop**
I made multiple small changes, resolving multiple (often old) issues. Can someone please do a review?
As the issues are unrelated to each other, I suggest reviewing by individual commits, not changed files. Individual commits also contain details and reasons on changes.

For testing, see a related PR in web-cz.

### CODE REVIEW

File: assets/_scss/core_design.scss [hunk 1]
```diff
@@ -78,6 +78,11 @@ p, li, dd {
     text-align: justify;
 }
 
+// Offset margin between paragraph and ul
+p+ul {
+    margin-top: -1rem;
```
Comment (mgrabovsky) [inline assets/_scss/core_design.scss:83]: I very much disagree with this change. It makes more sense to me to keep the list indented from the surrounding text, keeping the vertical rhythm and adding a bit of air in between the paragraphs.

The left column below looks a bit more balanced to me personally.

ul

I would even suggest a “counter-change”: Add the same space that's between paragraphs in between list items as well.

**Review by mgrabovsky — COMMENTED** [review]
I only take issue with one of the changes. Looks good otherwise.

File: assets/_scss/core_design.scss (hunk 1 above)
Comment (jankrcal) [reply assets/_scss/core_design.scss:83]: I agree with Matej, not feeling strongly about the counter-change, though. If adding space between bullet items, I'd strongly lean towards a smaller spacing (maybe half of the spacing for paragraphs?).

File: _includes/footer.html [hunk 2]
```diff
@@ -1,12 +1,26 @@
 <footer>
     <div class="container">
         <div>
+            {% if site.newsletter %}
             <a class="no-ext-link-icon" href="{{ site.newsletter }}">
-            <span class="fas fa-fw fa-envelope-open-text"></span> {{ site.data.lang.footer.get-newsletter }}</a><br/>
+                <span class="fas fa-fw fa-envelope-open-text"></span> {{ site.data.lang.footer.get-newsletter }}</a><br/>
```
Comment (jankrcal) [inline _includes/footer.html:6]: nit: why is this indented and the other instances not?

**Review by jankrcal — APPROVED** [review]
also looks good to me % minor comments

File: _includes/footer.html (hunk 2 above)
Comment (mukrop) [reply _includes/footer.html]: That was just a typo, fixed.

**Comment by mukrop** [comment]
Although I prefer no/smaller space at the beginning of the list, I'm not strong in that opinion. This PR was aimed mostly at no-brainers, so I dropped that change altogether.

PS: My rationale for no space is that often it's logically just a list of small things and thus still the same "logical block" as the introductory sentence (for example in the [updates page](https://faktaoklimatu.cz/aktuality)). I agree that if it's longer paragraphs in the bullets that are their separate logical block, the spaces should be above, below as well as between bullets. Maybe eventually having two styles for `ul` based on this distinction? Though not a priority right now,

**Pull request merged**